[2022-12-25T19:50:56.992+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-10-15T15:00:00+00:00 [queued]>
[2022-12-25T19:50:57.004+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-10-15T15:00:00+00:00 [queued]>
[2022-12-25T19:50:57.005+0300] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:50:57.005+0300] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2022-12-25T19:50:57.006+0300] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:50:57.022+0300] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): first_task> on 2022-10-15 15:00:00+00:00
[2022-12-25T19:50:57.027+0300] {standard_task_runner.py:55} INFO - Started process 33665 to run task
[2022-12-25T19:50:57.038+0300] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_pivot', 'first_task', 'scheduled__2022-10-15T15:00:00+00:00', '--job-id', '708', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_total.py', '--cfg-path', '/var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T/tmp9_fp4l3q']
[2022-12-25T19:50:57.041+0300] {standard_task_runner.py:83} INFO - Job 708: Subtask first_task
[2022-12-25T19:50:57.067+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:50:57.069+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_alchemy_conn] has been deprecated, you should use[database/sql_alchemy_conn] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:50:57.072+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:400 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:50:57.073+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_engine_encoding] has been deprecated, you should use[database/sql_engine_encoding] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:50:57.105+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/utils/sqlalchemy.py:121 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:50:57.150+0300] {task_command.py:389} INFO - Running <TaskInstance: titanic_pivot.first_task scheduled__2022-10-15T15:00:00+00:00 [running]> on host mbp-vadim.bbrouter
[2022-12-25T19:50:57.329+0300] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-10-15T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-15T15:00:00+00:00
[2022-12-25T19:50:57.330+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T
[2022-12-25T19:50:57.331+0300] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "Here we start!"']
[2022-12-25T19:50:57.343+0300] {subprocess.py:86} INFO - Output:
[2022-12-25T19:50:57.349+0300] {subprocess.py:93} INFO - Here we start!
[2022-12-25T19:50:57.350+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-25T19:50:57.388+0300] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=first_task, execution_date=20221015T150000, start_date=20221225T165056, end_date=20221225T165057
[2022-12-25T19:50:57.417+0300] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-25T19:50:57.459+0300] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
